modules/mobile/app/models/mobile/v0/appointment.rb

```diff
@@ -22,11 +22,12 @@ class Appointment < Common::Resource
         'VA_VIDEO_CONNECT_GFE',
         'VA_VIDEO_CONNECT_HOME'
       )
-      STATUS_TYPE = Types::String.enum('BOOKED', 'CANCELLED', 'HIDDEN')
+      STATUS_TYPE = Types::String.enum('BOOKED', 'CANCELLED', 'HIDDEN', 'SUBMITTED')
```

Comment:
Appointment requests can be either cancelled or submitted. "Submitted" isn't quite as clear as "proposed" but I stuck with the name because I thought it was better to stick with established names in other parts of the system. I think it makes sense when combined with `isPending`.